### PR TITLE
Change client attributes with client labels

### PR DESCRIPTION
**Pull request by ihsandemir**
Changed client attributes with client labels.

### CODE REVIEW

File: src/docs/asciidoc/hazelcast_clients.adoc [hunk 1]
```diff
@@ -304,13 +304,13 @@ The equivalent programmatic approach is shown below.
...
-clientConfig.setAttribute("role", "user");
-clientConfig.setAttribute("region", "bar");
+clientConfig.addLabel("user");
+clientConfig.addLabel("bar");
 
 HazelcastClient.newHazelcastClient(clientConfig);
 ----
 
-See the https://github.com/hazelcast/hazelcast-code-samples/tree/master/clients/client-attributes[code sample] for the client attributes to see them in action.
+See the https://github.com/hazelcast/hazelcast-code-samples/tree/master/clients/client-labels[code sample] for the client attributes to see them in action.
```
Comment (sancar) [inline src/docs/asciidoc/hazelcast_clients.adoc]: ... for the client labels to see them in action.

Comment (Serdaro) [reply src/docs/asciidoc/hazelcast_clients.adoc]: @ihsandemir , @sancar , the "attribute" usage in the reference manual will not create a confusion right?

Comment (ihsandemir) [reply src/docs/asciidoc/hazelcast_clients.adoc]: done

File: src/docs/asciidoc/hazelcast_clients.adoc [hunk 2]
```diff
@@ -277,23 +277,23 @@ if(lifecycleService.isRunning()){
...
 <hazelcast-client>
     ...    
     <instance-name>barClient</instance-name>
-    <client-attributes>
-        <attribute name="role">user</attribute>
-        <attribute name="region">bar</attribute>
-    </client-attributes>
+    <client-labels>
+        <label>user</attribute>
+        <alabel>bar</attribute>
```
Comment (sancar) [inline src/docs/asciidoc/hazelcast_clients.adoc]: There are typos here.

**Pull request merged**